### PR TITLE
fix(#2579): say and retry methods now work correctly

### PR DESCRIPTION
**Pull request by Vorobeyko**
## Motivation/Description of the PR
- Description of this PR, which problem it solves
- Resolves #issueId (if applicable).

Applicable helpers:

- [ ] WebDriver
- [ ] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] FileHelper
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe
- [ ] Playwright

Applicable plugins:

- [ ] allure
- [ ] autoDelay
- [ ] autoLogin
- [ ] customLocator
- [ ] pauseOnFail
- [ ] puppeteerCoverage
- [ ] retryFailedStep
- [ ] screenshotOnFail
- [ ] selenoid
- [ ] stepByStepReport
- [ ] wdio

## Type of change

- [ ] :fire: Breaking changes
- [ ] :rocket: New functionality
- [ ] :bug: Bug fix
- [ ] :clipboard: Documentation changes/updates
- [ ] :hotsprings: Hot fix
- [ ] :hammer: Markdown files fix - not related to source code
- [ ] :nail_care: Polish code

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [ ] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: lib/container.js [hunk 1]
```diff
@@ -331,6 +331,20 @@ function loadSupportObject(modulePath, supportObjectName) {
   }
   try {
     const obj = require(modulePath);
+
+    if (typeof obj === 'function' && supportObjectName === 'I') {
+      const fobj = obj();
+
+      if (fobj.constructor.name === 'Actor') {
+        const methods = getObjectMethods(fobj);
```
Comment (Georgegriff) [inline lib/container.js:339]: Wondering if is it 100% required that people implementing a steps file must use the `actor` object I seem to think that it's possible to implement a steps file without doing that i.e. just return an object, and if so if we've subtly breaking people with 3.0

Comment (Georgegriff) [reply lib/container.js:339]: Checked my code was wrong think maybe consumers using actor object is required like documented

**Review by Georgegriff — APPROVED** [review]

File: lib/container.js [hunk 2]
```diff
@@ -331,6 +331,20 @@ function loadSupportObject(modulePath, supportObjectName) {
   }
   try {
     const obj = require(modulePath);
+
+    if (typeof obj === 'function' && supportObjectName === 'I') {
```
Comment (DavertMik) [inline lib/container.js:335]:
```suggestion
    if (typeof obj === 'function') {
```
Do we need to hardcode `I` here?
I think not

File: lib/container.js (hunk 1 above)
Comment (DavertMik) [reply lib/container.js:339]: Good quetsion - technically yes, you can have anything. But without an actor, it doesn't have much sense

**Review by DavertMik — APPROVED** [review]

**Pull request merged**